quickstart/201-aks-log-analytics/main.tf

```diff
@@ -1,18 +1,8 @@
-# The Azure Active Resource Manager Terraform provider
-provider "azurerm" {
-  version = "=1.36.1"
-}
-
-# The Azure Active Directory Terraform provider
-provider "azuread" {
-  version = "=0.6.0"
-}
-
 # Reference to the current subscription.  Used when creating role assignments
 data "azurerm_subscription" "current" {}
 
 # The main resource group for this deployment
 resource "azurerm_resource_group" "default" {
   name     = "${var.name}-${var.environment}-rg"
```

Comment:
Please add a random string as resource group name's suffix since we might run the tests in parallel.

Reply:
Will do.